docs/developers.md

```diff
@@ -54,6 +54,14 @@ workflow for doing this is as follows:
 5. When you are ready for us to review, push your branch to GitHub, and
    then open a new pull request with us.
 
+### Third Party Dependencies
```

Comment:
Ive added an issue about updating labels (#1277) so when people submit prs that need dependency updates/additions we can track them.